gateway-server/src/test/java/org/apache/knox/gateway/config/impl/GatewayConfigImplTest.java

```diff
@@ -348,4 +348,43 @@ public void testCustomIdentityKeystoreOptions() {
     assertEquals("custom_keystore_password_alias", config.getSigningKeystorePasswordAlias());
   }
 
+  @Test
+  public void testHttpClientTruststoreOptions() {
+    GatewayConfigImpl config = new GatewayConfigImpl();
+
+    // Validate default options (backwards compatibility)
+    assertEquals("gateway-httpclient-truststore-password", config.getHttpClientTruststorePasswordAlias());
+    assertEquals("JKS", config.getHttpClientTruststoreType());
+    assertNull(config.getHttpClientTruststorePath());
+
+    // Validate changed options
+    config.set("gateway.httpclient.truststore.password.alias", "custom_password_alias");
```

Comment:
Can we use the gatewayconfig constants here?

Reply:
I didn't want to do that since I wanted to test that the constants (used internally) were correct.